src/clojure_tic_tac_toe/core.clj

```diff
@@ -1,16 +1,31 @@
 (ns clojure-tic-tac-toe.core
-  (:gen-class))
+  (:require [clojure-tic-tac-toe.input_output :as io]
+            [clojure-tic-tac-toe.board :as board]))
 
-(defn board-to-string []
-  (clojure.string/join "\n" [" 1 | 2 | 3 "
-                             "---+---+---"
-                             " 4 | 5 | 6 "
-                             "---+---+---"
-                             " 7 | 8 | 9 "]))
+(defn get-move
+  [player]
+  (io/prompt-player-for-move player)
+  (flush)
+  (io/get-move))
+
+(defn int-to-keyword
+  [integer]
+  (keyword (str integer)))
+
+(defn play-round
+  [board player]
+  (def move (get-move player))
```

Comment:
do you know about clojure `let`? https://clojuredocs.org/clojure.core/let
let might be more appropriate here instead of these defs

Reply:
Using `let` now.